src/main/kotlin/hr/foi/pop/backend/exceptions/ActivateUserException.kt

```diff
@@ -0,0 +1,12 @@
+package hr.foi.pop.backend.exceptions
+
+import hr.foi.pop.backend.definitions.ApplicationErrorType
+
+class ActivateUserException(private val error: ApplicationErrorType) : RuntimeException() {
+    override val message: String
+        get() = "$error: Check incoming user object!"
+
+    fun getApplicationErrorType(): ApplicationErrorType {
+        return this.error
+    }
```

Comment:
You don't need a getter - there's a reason why we stepped away from Java! 😄
Just remove a getter and make the constructor parameter public (by removing the `private` keyword).